Orange/preprocess/preprocess.py

```diff
@@ -569,6 +571,32 @@ def __call__(self, data):
             data = pp(data)
         return data
 
+class RemoveSparse(Preprocess):
+    """
+    Remove sparse  features. Sparseness is determined according to
+    user-defined treshold.
+
+    Parameters
+    ----------
+    threshold : float
+        Minimal proportion of non-zero entries of a feature
+    """
+
+    def __init__(self, threshold=0.05):
+        self.threshold = threshold
+
+    def __call__(self, data):
+        if sp.issparse(data.X):
+            data_csc = sp.csc_matrix(data.X)
+            h, w = data_csc.shape
+            sparsness = [data_csc[:, i].count_nonzero()/h for i in range(w)]
+        else:
+            sparsness = np.count_nonzero(data.X, axis=0)/ data.X.shape[0]
```

Comment:
Because you'll be rebasing and pushing anyway, you can also add a space before `/`, so it's balanced. Bonus points for putting spaces before and after `/` two lines above.